torch/_inductor/codegen/wrapper.py

```diff
@@ -573,7 +576,7 @@ def add_fake_input(name, shape, stride, device, dtype):
             )
 
     def define_kernel(self, name: str, kernel: str):
-        self.header.splice(f"\n\n{name} = {kernel}")
+        self.header.splice(f"{kernel}")
```

Comment:
This is a single kernel hack, for multi kernels, we need a couple lines of codegen for invoking K kernels from c++, which should be very easy to add.